node-notes-cli/app.js

```diff
@@ -0,0 +1,75 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+// Reading data from data.json
+async function readNotes() {
+  const jsonData = await readFile('./data.json', 'utf8');
+  const data = JSON.parse(jsonData);
+  const notes = [];
+
+  for (const note in data.notes) {
+    notes.push(`${note}: ${data.notes[note]}`);
+  }
+  notes.forEach((note) => console.log(note));
```

Comment:
I don't see the value in putting the notes into a separate `notes` array. Why not just log them directly in the for-loop?

Reply:
Updated not sure why I did that either, still trying to get myself comfortable with objects